_commands/tools/emacs.md

```diff
@@ -0,0 +1,80 @@
+---
+---
+
+emacs
+--
+
+`emacs` is used to open a file in emacs
+~~~ bash
+$ emacs filename
+~~~
+Then file specified by `filename` will be opened in emacs, and you can use emacs to edit it.
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `emacs -q`
+
+~~~ bash
+$ emacs -q
+~~~
+
+##### Break it down
+
+ * `-q` means do not load an init file.
+ * `-q` will be ignored if it is followed by a `filename`.
+
+#### `emacs +number file`
+
+~~~ bash
+$ emacs +7 filename
+~~~
+
+##### Break it down
+ * Go to the line specified by `number`.
+ * Do not insert a space between the "+" sign and the `number`.
+
+#### `emacs -font, -fn`
+
+~~~ bash
+$ emacs -font 12 filename
+~~~
+
+~~~ bash
+$ emacs -fn 12 filename
+~~~
+
+##### Break it down
+
+ * Set the Emacs window’s font to that specified by font. You will find the various X fonts
+   in the /usr/lib/X11/fonts directory. Note that Emacs will only accept fixed width fonts. 
+   Under the X11 Release 4 font-naming conventions, any font with the value "m" or "c" in the 
+   eleventh field of the font name is a fixed width font. Furthermore, fonts whose name are of
+   the form widthxheight are generally fixed width, as is the font fixed. See **xlsfonts**(1) for 
+   more information.
```

Comment:
This looks copied from a man page. The goal isn't to copy man documentation. How much of this paragraph is useful for the average c4cs student? The idea is to pull out the things that are actually useful, in this case something closer to "The font must be installed on your system to work. Some fonts installed that I like are [...]. For all the details on fonts, check out the `xlsfonts` manpage"